lib/NativeCode/DynamicLibraryLoaderHelper/NativeRender/dllmain.cpp

```diff
@@ -153,6 +153,7 @@ struct EOSConfig
     uint64_t flags = 0;
 
     uint32_t tickBudgetInMilliseconds = 0;
+    double taskNetworkTimeoutSeconds = 0;
```

Comment:
Should probably be `0.0`

Reply:
Changed! Interestingly the compiler already treated this as `0.0`, I can see from hovering over it.